diagrams.md

```diff
@@ -497,6 +497,7 @@ erDiagram
         Choice service_offering_group "ADVISORY_ASSISTANCE/APPLICATIONS/COMPUTE/DATABASE/DEVELOPER_TOOLS/DOCUMENTATION_SUPPORT/EDGE_COMPUTING/GENERAL_CLOUD_SUPPORT/GENERAL_XAAS/HELP_DESK_SERVICES/IOT/MACHINE_LEARNING/NETWORKING/PORTABILITY_PLAN/SECURITY/STORAGE/TRAINING"
         Integer sequence
         String name
+        Choice offering_type "XAAS_SERVICE/CLOUD_SUPPORT"
```

Comment:
Not sure if `dow_task_number` was left out intentionally or not, but should that be included in the diagram for tables updated like `TRAINING_ESTIMATE`?

Reply:
If I understand your question, I believe [this commit](https://github.com/dod-ccpo/atat-snow/pull/307/commits/38786895e3d0667a1d81487b1acc3ea5d1adb01f) added those columns to the documentation.
#307 should have all of the columns that front-end will need to store numbers per Tony.

Reply:
Oh okay, I completely missed those in the documentation 😆